src/commands/api/add-ext.ts

```diff
@@ -0,0 +1,57 @@
+import { CliUx, Command, Flags } from '@oclif/core'
+import { ApiType, ExternalApiClaim } from '../../models/bundle-descriptor'
+import { ApiClaimService } from '../../services/api-claim-service'
+import { BundleService } from '../../services/bundle-service'
+
+export default class AddExt extends Command {
+  static description =
+    'Adds an external API claim to the specified MFE component'
+
+  static examples = [
+    '$ <%= config.bin %> <%= command.id %> mfe1 ms1-api --bundleId my-bundle --serviceId ms1'
```

Comment:
It looks that this `$` can be removed, indeed in the other commands we are not adding it and it is automatically added by Oclif.